packages/clisp.rb

```diff
@@ -1,40 +1,32 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Clisp < Package
+class Clisp < Autotools
   description 'ANSI Common Lisp compiler, interpreter and debugger.'
-  homepage 'http://www.gnu.org/software/clisp/'
-  version '2.49-3'
-  license 'GPL-2'
-  compatibility 'i686 x86_64'
-  source_url 'https://ftpmirror.gnu.org/clisp/release/2.49/clisp-2.49.tar.bz2'
-  source_sha256 '8132ff353afaa70e6b19367a25ae3d5a43627279c25647c220641fed00f8e890'
-  binary_compression 'tar.xz'
+  homepage 'https://www.gnu.org/software/clisp/'
+  version '2.49-c3ec11b'
```

Comment:
Won't this mess with repology expected versions?

Reply:
We trim git hashtags before writing to repology.json, so this will just show up as 2.49 there.

Reply:
Great, just checking...